Bottom: 6885619ee8103b0fb68a4200b3f167b846e195db
Top:    fab2f37fe30e60471f1000348075b2a46c5f0c85
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 20:31:48 +0000

Mark segments as not round before falling back to line-line intersection

This avoids assert failures with debugging enabeled


---

diff --git a/src/polygon1.c b/src/polygon1.c
index f7d39c6..f8c1d48 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1292,6 +1292,8 @@ seg_in_seg_arc_arc (struct info *i, struct seg *s1, struct seg *s2)
       s1->v->radius == s2->v->radius)
     {
       printf ("Cop-out for co-circular arcs for now - they will be better handled by their line approximations. <<<<<<<<<<\n");
+      s1->v->is_round = false;
+      s2->v->is_round = false;
       return seg_in_seg_line_line (i, s1, s2);
     }
 #endif
